### ISSUE TITLE
Multiple describe blocks are not supported for IE11 browser report.html

### ISSUE THREAD

**Issue by Tehaseen** [Evilweed/protractor-beautiful-reporter#31]
**Observation:**
Multiple describe blocks are not supported for IE11 browser report.html

The browser works fine till the second describe , after 2nd describe block it becomes unresponsive and give **Error 404** or it gives **'could not store metadata or add Metadata to combine.js protractor'** error

i.e
**describe('I am the first describe block', function(){

      describe('I am the second describe block', function(){
     // works fine till the second describe block
       });
  
     describe('I am the third describe block', function(){
     // from the third describe block , the page becomes unresponsive    
      });

} );**

I have attached a test Calculator script where i have same issue
[Calculator.zip](https://github.com/Evilweed/protractor-beautiful-reporter/files/1562558/Calculator.zip)
Please run the conf file for reports to get generated.

**NOTE:** The same scenario works fine for **_Chrome_** Runs . we get a clean green run.

**Exepected result :** All the describe blocks should execute without any error .Ideally it should behave same as chrome runs

Many Thanks in Advance

**Comment by Evilweed** [comment]
@Tehaseen sadly i don't have any Windows machine i could use to debug this issue.

If someone else with Windows machine could deug it - i would be grateful.

**Comment by Evilweed** [comment]
@DortmunderJunge could You look into this? I don't have Windows machine to check it :<

**Comment by DortmunderJunge** [comment]
@Evilweed yes, actually we have the same scenario in our project and it is working totally fine. But I will take a look at @Tehaseen's example, maybe I can spot any difference.

The only thing I found not working is this:
```javascript
describe('Suite 1', () => {
  it('Spec 1', () => {
    expect(something).toBeTruthy();
    describe('Some weirdly nested Suite', () => {
      //more specs
    });
  });
  it('Spec 2', () => {
     //whatever
  });
});
```

Spec 1 and Spec 2 get executed correctly, but the specs in the nested describe-block do not execute (or at least do not appear in the report). I am not sure if this has something to do with the reporter, IE, or if this is related to protractor/jasmine itself. Plus, nesting ```describe``` in ```it``` is not a good pattern at all, I guess, so I ended up removing these pieces of code.
Now my nested describes work all good.

**Comment by DortmunderJunge** [comment]
Okay, I tried to reproduce this and here is what i found:

- The example does not include any package.json, so I have no clue which protractor-version or ie-driver-version @Tehaseen is using. I ended up using the current default versions (protractor@5.3.0, IE-Driver@3.9.0)
- Chrome produces 3/4 green tests, test number 2 fails with ```expected '200' to equal 200```
- IE fails for all tests but the first
- BUT: IE executes all tests, there is just some bug that clicking the 'go'-Button does not work, so the calculator does not calculate anything and all failures are caused by falsy expectations like ```expected '0' to equal '100'``` which is totally legit here
- The example is a little weird, Promises from within the page-objects are not returned and might thus not be awaited and conf_IE.js does not set the path to the IE-Webdriver into the ```localSeleniumStandaloneOpts.jvmArgs``` property. This might cause some unexpected behavior.

I even executed the tests without generating a report (deleting the ```protractor-beautiful-reporter``` from node_modules to be sure) and ended up with the same test-result in my console.

**Conclusion:**
Although these tests fail in IE, I do not see any reason why it might be related to this project. I suspect some general protractor-related-error within @Tehaseen's example
